readme.txt

```diff
@@ -109,6 +109,17 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.10.0 =
+
+**Release Date:** July 13, 2021.
```

Comment:
Update to Aug 10, 2021?

Reply:
Whoops. Indeed.

That, or I just travelled through time 🤯